BEIMA.Backend.FT/TestObjects.cs

```diff
@@ -143,6 +143,27 @@ public class Building
             public LastModified? LastModified { get; set; }
         }
 
+        public class User
```

Comment:
I think we also need a LastModified property here.

Reply:
Fixed 57c5fac